assets/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "mayflower-theme",
```

Comment:
Is this a theme or is it more accurate to say this is a package of styles and static assets? It could be confusing to name this `mayflower-theme` if there becomes a `mayflower-uswds-theme` for example. This might not be the right PR for this discussion though, so feel free to ignore this comment.

Reply:
ok, let's table this for the next PR, and see which package name makes more sense `mayflower-styles`, `mayflower-assets`, `mayflower-styles-and-assets`?